terraform/env/monitoring.tf

```diff
@@ -0,0 +1,19 @@
+module "alerting" {
+  source = "../modules/alerting"
+
+  sns_general_availability_topic_name = "${var.sns_general_availability_topic_name}"
+  general_availability_protocol = "${var.general_availability_protocol}"
+  general_availability_endpoint = "${var.general_availability_endpoint}"
+}
```

Comment:
Oops, I guess this should be in `mgmt`?

Reply:
In my mind, this would be part of the bootstrap environment, but could go either way. Since the SNS is a resource that isn't tied to a particular VPC and both environments use it, I thought maybe it should go in bootstrap. But it could be argued either way, that's just how I usually think. ¯\_(ツ)_/¯

Reply:
I was keeping `bootstrap` to the bucket for the Terraform backend. Since the state is only stored locally, I wouldn't want to put more in that module. Maybe the name `bootstrap` is misleading?

Reply:
Anyway, if we only want one of them, we just keep the one under `mgmt` and delete this block.